### PR TITLE
is.gsplot() function

### PR DESCRIPTION
**Pull request by lindsayplatt**
Not integrated with `print` or anything - just works solo and returns a vector saying "Valid" or "Invalid" for each check

### CODE REVIEW

File: R/is_gsplot.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    class <- "Valid"	
+    
+    if (!"par" %in% duplicates) {
+      par_arguments <- "Valid"
+    } else { par_arguments <- "Invalid" }
+    
+  } else { class <- "Invalid" }
+  
+  validity <- rbind(class, par_arguments)
+  return(validity)
```
Comment (ldecicco-USGS) [inline R/is_gsplot.R:45]: We'll want the return to be either TRUE or FALSE.

Comment (lindsayplatt) [inline R/is_gsplot.R:45]: So one sweeping statement -- it's either all valid (TRUE) or at least one thing is invalid (FALSE)

Comment (ldecicco-USGS) [inline R/is_gsplot.R:45]: Yup. These type of functions are usually used like (if(is.gsplot(x)){blah...)}

Comment (lindsayplatt) [inline R/is_gsplot.R:45]: Ok, right - I've used `is.numeric` and stuff. I should have thought of that! Thanks :smile: new PR is in

**Pull request closed without merging**